web/test/pages/onboarding/onboarding-starting.test.tsx

```diff
@@ -274,4 +274,13 @@ describe("onboarding - starting a business", () => {
   describe("validates self-reg step", () => {
     runSelfRegPageTests({ businessPersona: "STARTING", selfRegPage: "4" });
   });
+
+  describe("when onboarding flow is starting", () => {
```

Comment:
this test name is a little confusing because it's not really testing when the "onboarding flow" is starting. Instead, it's testing when the _query parameter_ flow is starting. These are different things, because the query parameter flow actually _sets_ the onboarding flow. So saying that the description of the test is "_when_ the onboarding flow is starting" is kind of backwards